modules/aws-outposts-prereq-storage-csi.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-prereq-storage-csi_{context}"]
+= Preparing the CSI driver to support AWS Outposts
+
+You must update the Container Storage Interface (CSI) driver default storage class to be compatible with AWS Outposts.
+To prevent the CSI Operator from recreating the initial default storage class, you must set its state to *Unmanaged*.
+
+The CSI driver uses the topology keys stored on the `CSINode` objects to determine the Amazon Resource Name (ARN) for the Outpost.
+The driver requires the ARN to create volumes in an AWS Outposts instance.
+
+{product-title} clusters on AWS include the `gp3-csi` and `gp2-csi` storage classes.
+These classes correspond to Amazon Elastic Block Store (EBS) gp3 and gp2 volumes.
+{product-title} clusters use the `gp3-csi` storage class by default, but AWS Outposts does not support EBS gp3 volumes.
+
+.Prerequisites
+
+* You have installed an {product-title} cluster into a custom VPC on AWS.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Get the storage driver provisioner name by running the following command:
++
+[source,terminal]
+----
+$ oc get storageclass
+----
++
+.Example output
+[source,text]
+----
+NAME                PROVISIONER       RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+gp2-csi             ebs.csi.aws.com   Delete          WaitForFirstConsumer   true                   7h43m
+gp3-csi (default)   ebs.csi.aws.com   Delete          WaitForFirstConsumer   true                   7h43m
+----
++
+In this example, the provisioner name is `ebs.csi.aws.com`.
+
+. Set the the `spec.storageClassState` field in the `ClusterCSIDriver` object to `Unmanaged` by running the following command:
++
+[source,terminal]
+----
+$ oc patch clustercsidriver <provisioner_name> \
+  --type=merge \
+  -p "{\"spec\":{\"storageClassState\":\"Unmanaged\"}}"
+----
++
+where `<provisioner_name>` is the value of `PROVISIONER` in the `oc get storageclass` command output.
+
+. Set the default storage class to use gp2 volumes by running the following commands:
```

Comment:
Changing the default SC is covered in this topic: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/storage/using-container-storage-interface-csi#change-default-storage-class_persistent-storage-csi-sc-manage